src/content/get-started/overview.mdx

```diff
@@ -16,31 +16,37 @@ If you are interested in using Okteto's SaaS offering, you can ignore this page
 
 In order to fully install Okteto, you'll need the following:
 
-- A domain to which you can add a wildcard DNS record
 - A Kubernetes cluster
+- An Okteto License
+- A domain to which you can add a wildcard DNS record
```

Comment:
I've always found this wording confusing.  You don't add records to a domain. Maybe we just leave it as: "a domain and the ability to create DNS records for it"?